### ISSUE TITLE
[config request I think] remove/turn off ring of Odin

### ISSUE THREAD

**Issue by OakCobra** [ChloeTax/hexxytest3#36]
Ok so a summary of my point it that on this server we have many many mod that are all interesting and intriguing in their own ways and all have cool ways to fill certain niches, but botania has one items that is an egregious offender to this, now the issue is NOT that these ability’s exist, I 100% support these being able to be achieved, but what the issue is is that botania Odin run provides complete immunity to most all damage types and gives a very large health boost in a very very easy to get and braindead package, this requires little effort of the player to get or more importantly little mental effort and very little thinking to get, essentially Odin ring= crazy simple and braindead way to get a ton of things that other mods we have can provide which almost optimizes the fun out of achieving these things. And once one person gets it nearly everyone else will get it since it’s so easy to get, optimizing fun away.

ALSO a lot of people are saying oh nobody will use it, you’re wrong. So if nobody uses it then what’s the issue with Turing it off?

**Comment by Twisted-Code** [comment]
Someone also suggested possibly making RoO a health boost only, no environmental defenses..

**Comment by OakCobra** [comment]
Yes I think that would also be an acceptable fix, although maybe only like 5 health boost (just my opinion, amount doesn’t bother me too much)

**Comment by Twisted-Code** [comment]
Also, I raise a personal objection to the claim that Botania relics are "very very easy" to get. Technically, they require the following:
- Most/all of Botania's "main track" progression infrastructure (runic altar, TAP, the portal to Alfheim...)
- Lots of mana for crafting up to this point
- one terrasteel per relic, there goes another half a pool of mana
- some way to actually fight the thing
I think if you consider it, the real problem isn't that this item is NECESSARILY too powerful, but that people have optimized all of the dependencies to a very noticeable degree. I don't think I need to elaborate on what these methods are, they are ubiquitous enough at this point, right?
Botania balance gets better, I think, when the it is used in isolation, but even then it has been changed so little through the updates that even the most challenge oriented players may have trouble avoiding optimization spoilers.
All of that to say: I tentatively agree it could use some balance tweaking to fit better in context, but I think you are oversimplifying the dynamic that led us to this situation.

**Comment by Twisted-Code** [comment]
So like I said, we've optimized how to get there...

**Comment by OakCobra** [comment]
Brought??

also I still think it would be beneficial to server longevity and enjoy ability if it was turned off so I think it should be removed, or turned into just a max health boost

**Comment by OakCobra** [comment]
Oh, I think we should go about removing it unless someone decides to make something to change it and just give the 3 people the option to choose something new

**Comment by Twisted-Code** [comment]
Great, now we have no relics at all :-/

**Comment by Stick404** [comment]
I understand limiting the ring, but turning all of them off limits Botania's endgame.
